typescript/quantize/lab_point_provider.ts

```diff
@@ -15,9 +15,9 @@
  * limitations under the License.
  */
 
-import './point_provider';
+import './point_provider.js';
```

Comment:
typescript/quantize/lab_point_provider.ts:18 - `.js` extension added in this PR but not added in internal change.